src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const carCost = 40;
+  let allCost = days * carCost;
+
+  if (days >= 7) {
+    allCost -= 50;
+
+    return allCost;
+  }
+
+  if (days >= 3) {
+    allCost -= 20;
+
+    return allCost;
+  }
```

Comment:
[Checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md) 3rd item, you can return value immediately
```suggestion
  if (days >= 7) {
    return allCost - 50;
  }

  if (days >= 3) {
    ... same here
  }
```